site/content/entry/yuzu-progress-report-dec-2022/index.md

```diff
@@ -0,0 +1,399 @@
+lioncash also made the {{< gh-hovercard "9394" "SPIR-V shader backend element translatable," >}} so it doesn’t always show in English for everyone.
+The community effort working on translation can now take the label and update it accordingly.
+
+{{< imgs
+	"./spirv.png| We still don’t recommend using it over GLSL, but Mesa users report they enjoy it"
+  >}}
+
+Some months ago, with the core timing changes, we allowed users to boot games with their framerate unlocked, after continuous requests by the community.
+As it turns out, nothing changed, several games hate booting with unlocked framerates, and the support channels get their fair share of people asking why their game doesn’t want to boot.
+So, simple fix, {{< gh-hovercard "9425" "unlocked framerate at boot rights denied." >}}
+The hotkey is Ctrl + U by default.
+
+Users reported that they couldn’t record or stream their yuzu window while in windowed mode.
+byte[] found the cause was setting the `WA_DontCreateNativeAncestors` Qt property for all platforms, instead of just for wayland.
+{{< gh-hovercard "9461" "Issue down, streamers rejoyce." >}}
+
+Discord user piplup (with a very cute profile picture, I have to add) reported that yuzu didn’t save the device name (what you would call the console) after accessing a game’s custom configuration window.
+german77 {{< gh-hovercard "9466" "fixed the issue" >}} (this particular setting lacked a custom configuration equivalent), and also fixed Qt 6 build issues while at it.
+
+Another very nice quality of life improvement made by german77 is making yuzu {{< gh-hovercard "9467" "remember the last selected directory" >}} for `Install files to NAND…`.
+If you keep your dumps in the same folder, updating your games is going to take fewer clicks now.
+
+byte[] managed an amazing victory in the war against crashes when closing/stopping games. He worked on {{< gh-hovercard "9476" "making shutdown not visibly freeze yuzu," >}} avoiding crashing the emulator while the game quits, and showing a nice pop-up message while at it too!
+
+{{< imgs
+	"./shutdown.png| We take longer than the Switch, but some games really love to take their time on console"
+  >}}
+
+[Morph](https://github.com/Morph1984) helped make this possible by {{< gh-hovercard "9477" "hiding button dialog boxes," >}} allowing to create the dialog overlay byte[] added.
+
+Another battle fought in this front is related to homebrew apps. byte[] is responsible for {{< gh-hovercard "9486" "making them quit properly" >}} now too.
+
+[ChrisOboe](https://github.com/ChrisOboe) suddenly shows up with a glorious quality of life fix for the terminal-based yuzu-cmd build.
+Marking the build as a “Windows” application instead of a “Console” one {{< gh-hovercard "9485" "ensures that no empty command line window pops up" >}} needlessly.
+This can help streaming programs set up to run specific games with yuzu-cmd, as there is no sudden empty black box bothering at the front now.
+Thanks!
```

Comment:
```suggestion
[ChrisOboe](https://github.com/ChrisOboe) suddenly shows up with a glorious quality of life fix for the terminal-based yuzu-cmd build.
Marking the build as a “Windows” application instead of a “Console” one {{< gh-hovercard "9485" "ensures that no empty command line window pops up" >}} needlessly.
This can help streaming programs set up to run specific games with yuzu-cmd, as this prevents the sudden empty black box from appearing in front of other windows.

```